printer_mods/JosAr/Klicky-Probe/Klipper_macro/README.md

```diff
@@ -0,0 +1,8 @@
+Voron V2.4:
+If using the probe instead of a physical Z endstop please remember to comment out the homing overide from your printer.cfg and use the klicky-probe.cfg one.
+For using the Klicky probe as a probe only it is not necessary to remove anything from your existent configuration, it is better to comment the homing overide on klicky-probe.cfg
+
+Voron V1.8:
+Due to the fact that the probe may hit the bed on the v1.8 when going to the physical Z endstop, please use the homing overide included in klicky-probe.cfg instead of the one that you currently have on your printer.cfg.
```

Comment:
Replace `overide` with `override`

Replace `on your printer.cfg` with `in your printer.cfg`.